hw/Makefile

```diff
@@ -10,27 +10,25 @@ REG_OUTPUT_DV_DIR ?= ${REG_OUTPUT_DIR}/dv
 REG_OUTPUT_SW_DIR ?= ${REG_OUTPUT_DIR}/sw
 REG_OUTPUT_AUTOGEN_DIR ?= ${PRJ_DIR}/build/autogen
 
+# These are the IPs whose reg collateral is not generated and have no other
+# collateral generation.
```

Comment:
We may want to say that the reg collateral is not generated by `ipgen`, because it will be "generated" via `reggen` if listed here.

Reply:
The non-ipgen generated blocks (like flash_ctrl) also have reg collateral generated by topgen. I clarifed the comment.